vendor/k8s.io/kubernetes/hack/openapi-violation.list

```diff
@@ -1,497 +1,3 @@
-API rule violation: list_type_missing,k8s.io/api/admissionregistration/v1,MutatingWebhook,AdmissionReviewVersions
```

Comment:
@bparees @njhale Do you know how to solve these vendor issues? I just run `./hack/update-generated-openapi.sh` command.